src/public-lib/components/WindowWrapper.ts

```diff
@@ -10,15 +10,19 @@ export interface WindowSize {
 export default class WindowWrapper
 {
     @observable public size: WindowSize;
+    @observable public size500Ms: WindowSize;
 
     private handleWindowResize = _.debounce(this.setWindowSize, 200);
+    private handleWindowResize500Ms = _.debounce(this.setWindowSize500Ms, 500);
```

Comment:
attention here. is this okay? I added this because for something really expensive to rerender, like oncoprint, its better to debounce it even more